src/Graphics/Context.h

```diff
@@ -17,12 +17,13 @@ namespace graphics {
 
 	enum class SpecialFeatures {
 		Multisampling,
-		FragmentDepthWrite,
 		BlitFramebuffer,
 		WeakBlitFramebuffer,
 		DepthFramebufferTextures,
 		ShaderProgramBinary,
-		ImageTextures
+		ImageTextures,
+		LUTTextures,
```

Comment:
LUTTextures is a bad name. Can you pick more general name, which reflect essence of that special feature?